app/livechat/server/lib/stream/agentStatus.ts

```diff
@@ -17,17 +17,8 @@ settings.get('Livechat_agent_leave_action_timeout', (_key, value) => {
 });
 
 settings.get('Livechat_agent_leave_action', (_key, value) => {
-	if (typeof value !== 'boolean') {
-		return;
-	}
-	monitorAgents = value;
-});
-
-settings.get('Livechat_agent_leave_action', (_key, value) => {
-	if (typeof value !== 'string') {
-		return;
-	}
-	action = value;
+	monitorAgents = value !== 'none';
+	action = value as string;
```

Comment:
you don't need this type casting right?
```suggestion
	action = value;
```

Reply:
it is actually needed 🤦